lib/index.js

```diff
@@ -18,6 +18,17 @@ var render = require('posthtml-render')
  * @param {Array} plugins - An array of PostHTML plugins
  */
 function PostHTML (plugins) {
+  /**
+ * PostHTML Instance
+ *
+ * @name posthtml
+ * @version 0.9.1
+ *
+ * @prop plugins
+ * @prop options
+ */
+  this.version = '0.9.1'
```

Comment:
Modules are cached after the first time they are loaded. But before return an object - Node.js will calculate the full path (module id) to `require()`. For more details, we can trace the call chain:

  1. https://github.com/nodejs/node/blob/master/lib/module.js#L498
  2. https://github.com/nodejs/node/blob/master/lib/module.js#L418
  3. https://github.com/nodejs/node/blob/master/lib/module.js#L461
  4. https://github.com/nodejs/node/blob/master/lib/module.js#L468
  5. etc.

P.S. It's a small delay.